stdlib/list.ml

```diff
@@ -56,6 +56,21 @@ let rec rev_append l1 l2 =
 
 let rev l = rev_append l []
 
+let rec init_tailrec_aux acc i n f =
+  if i >= n then acc
+  else init_tailrec_aux (f i::acc) (i+1) n f
+
+let rec init_aux i n f =
+  if i >= n then []
+  else
+    let fi = f i in
+    fi::init_aux (i+1) n f
```

Comment:
Again, for consistency elsewhere could variable `fi` be `r`, please (and spaces around the `::`)